src/common/containers/Header.jsx

```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faBars } from "@fortawesome/free-solid-svg-icons";
+import { signout } from "utils/signout";
```

Comment:
* Let's give the function a more appropriate name. For example, `signOut`.
* The utility is better called "users". So that we can add user-related features to it in the future.